README.md

```diff
@@ -1,10 +1,15 @@
 # AWS IoT Device SDK for Java v2
 
-This document provides information about the AWS IoT device SDK for Java V2. This SDK is built on the [AWS Common Runtime](https://docs.aws.amazon.com/sdkref/latest/guide/common-runtime.html)
+The **AWS IoT Device SDK for Java v2** is a rewrite of SDK v1 with some great new features. It includes many updates,
```

Comment:
I would like to start with a general overview of the SDK before diving into detailed features like “offline operation queue control.” 

For example: (generated from the DeepWiki page):
> The SDK enables you to connect IoT devices to AWS IoT Core, publish messages to topics, subscribe to topics, and use various AWS IoT services such as Device Shadow, Jobs, and Fleet provisioning.